services/debug/debug.service.js

```diff
@@ -35,6 +36,22 @@ module.exports = class Debug extends NonMemoryCachingBaseService {
           label: 'start time',
           message: new Date(serverStartTime).toUTCString(),
         }
+      // For production cache debugging.
+      case 'flip':
```

Comment:
I didn't even know these existed! I assume based on the inline context that these are solely for our usage/consumption, and not something an end user would have a need to consume, nor would this service by referenced by a contributor?

Reply:
Yup, exactly.

Reply:
In that case it LGTM. I was going to ask about the lack of a `render` function but it makes more sense now given the purpose 👍